google/cloud/firestore_v1/async_aggregation.py

```diff
@@ -129,7 +129,7 @@ def stream(
         transaction: Optional[transaction.Transaction] = None,
         retry: Optional[retries.AsyncRetry] = gapic_v1.method.DEFAULT,
         timeout: Optional[float] = None,
-    ) -> async_stream_iterator.AsyncStreamIterator:
+    ) -> "AsyncStreamGenerator[DocumentSnapshot]":
```

Comment:
Do you need quotes here? It looks like AsyncStreamGenerator is imported

Reply:
I added the quotes because `[DocumentSnapshot]` part would cause an error, because `AsyncStreamGenerator` isn't recognized as a generator in type hints. I wasn't able to find how to do it, so I resorted to using a string. I don't have a strong opinion on this. What do you think?